packages/cli/src/tools/releaseChecker/printNewRelease.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @flow
+ */
+import chalk from 'chalk';
+import logger from '../logger';
+import type {Release} from './getLatestRelease';
+import cacheManager from './releaseCacheManager';
+
+/**
+ * Notifies the user that a newer version of React Native is available.
+ */
+export default function printNewRelease(
+  name: string,
+  latestRelease: Release,
+  currentVersion: string,
+) {
+  logger.info(
+    `React Native v${
+      latestRelease.version
+    } is now available (your project is running on v${currentVersion}).`,
```

Comment:
Adjusted the message a bit, looks like this now:

![image](https://user-images.githubusercontent.com/5106466/59030958-b8327d80-8862-11e9-955f-a1d1ba569d42.png)

Hope you don't mind :)

Reply:
cc @cpojer is this ok, or do you have  better ideas?

Reply:
Not at all, I like this version more!